### ISSUE TITLE
Flow Metrics: If loss is not enabled for flow, better to skip returning loss at all instead of returning loss:null

### ISSUE THREAD

**Issue by ANISH-GOTTAPU** [open-traffic-generator/ixnetwork#372]
If loss is not enabled for flow, better to skip returning loss at all instead of returning loss:null

flow_metrics:
- bytes_rx: 612253696
  bytes_tx: 0
  frames_rx: 597904
  frames_rx_rate: 59865.5
  frames_tx: 597905
  frames_tx_rate: 59866.0
  `loss: null`
  name: f1
  port_rx: rx
  port_tx: tx
  transmit: started

**Comment by alakendu** [comment]
Could you please enable loss (flow.metrics.loss = True) and validate.

We will discuss what should be the behavior when flow.metrics.loss = False. Actually loss present in FlowMetric constructor. So loss will add automatically at the time of deserialize.

**Comment by ANISH-GOTTAPU** [comment]
if `flow.metrics.loss = True` , the loss is returned as expected which is  a reflection from GUI

**Comment by alakendu** [comment]
Whatever I have remember we are ok with this behavior @ANISH-GOTTAPU could you please double check and close it accordingly.

**Comment by ANISH-GOTTAPU** [comment]
closed
